src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -117,15 +117,20 @@ public void Configure(string name, OpenIdConnectOptions options)
 
         private async Task<OpenIdClientSettings> GetClientSettingsAsync()
         {
-            var settings = await _clientService.GetSettingsAsync();
-            if ((await _clientService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
+            var settings = await _clientService.GetSettingsAsync(); 
+            if (_logger.IsEnabled(LogLevel.Warning))
             {
-                if (_shellSettings.State == TenantState.Running)
+                var result = await _clientService.ValidateSettingsAsync(settings);
```

Comment:
This doesn't look right. Behaviour looks like it is supposed to be return null if they are invalid, so they'll still need to be validated, then it's just the `_logger.IsEnabled` to decide whether to compute and log the result.

Correct me if I'm wrong